documents/data-sharing.md

```diff
@@ -0,0 +1,58 @@
+# Annual survey data sharing
+
+This document describes how data from the annual survey should be shared.
+
+## Objectives
+
+In priority order:
+
+* Protect the privacy of respondents.
+* Provide high quality data (unbiased data which is unlikely to be misinterpreted, c.f., raw data).
+* Provide useful and actionable data to the Rust project (Core team, Foundation, teams and working groups, contributors).
+* Provide useful data for users and potential users of Rust and providers of Rust tools and services.
+* Provide data in a timely manner.
+* Provide marketing for Rust.
+* Avoid overloading people working directly with the survey data (note that there were over 9000 completed surveys and multiple opportunities for free form responses).
+
+Non-goals:
+
+* We will not aim to make money by selling the data in any form.
+* We will aim to not privilege companies which employ people with access to survey data (raw or processed). However, we acknowledge that controlling widely shared data is impossible.
```

Comment:
I might s/We/The project/ or similar just to make clear who is providing this (non)goal.